### ISSUE TITLE
[Resources & Support] Convert "Need more help?" custom HTML component to DS web component

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#19443]
## Description

On Resources & Support pages, we have a "Need more help?" section at the bottom of the page with contact phone numbers.

Example pages:
- https://www.va.gov/resources/evidence-to-support-va-pension-dic-or-accrued-benefits-claims/
- https://www.va.gov/resources/deciding-how-much-life-insurance-to-get/
- https://www.va.gov/resources/helpful-va-phone-numbers/
- https://www.va.gov/resources/how-to-get-help-with-concerns-at-a-va-health-facility/

That component is a custom HTML component in **content-build**: [contact_information.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/paragraphs/contact_information.drupal.liquid).

It has also been moved over into **next-build**: [templates/components/contactInfo](https://github.com/department-of-veterans-affairs/next-build/tree/main/src/templates/components/contactInfo)

If possible, we should convert this custom HTML component to a `Need help?` Design System web component. Docs here: https://design.va.gov/storybook/?path=/docs/components-va-need-help--docs

**Note**: The DS web component is currently marked as `USE WITH CAUTION: CANDIDATE` as of 10/8/24. We should verify that this component is ready to use before picking up this ticket to work.

<img width="500" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM013Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7344bc3d416a21c287b4b9a019907fb8490cfe73/Screenshot%202024-10-08%20at%2011.32.27%20AM.png" alt="Screenshot 2024-10-08 at 11.32.27 AM.png" />

Slack thread for context: https://dsva.slack.com/archives/C52CL1PKQ/p1727375505418149

### User story
> _Describe the audience/user, enhancement or fix, and value / outcome desired._

**AS A**
**I WANT**
**SO THAT**

### Engineering notes / background
This custom HTML component handles a variety of content such as:
- One default contact number
- Multiple numbers (in a list)
- A lot of conditional logic about which phone numbers to display and how to format them

If any of the above functionality is lost, we need to **keep the custom HTML component as-is** and report any gaps in functionality to DST (#platform-design-system in Slack) if improvements can be made to the web component.

## Testing & QA
Testing of the example URLs at minimum should be conducted. To identify test cases, review the logic in the code and be sure an example URL is found for every path the code can take. Regression testing for the rest of the R&S page should also be conducted to ensure that no other components broke or are no longer appearing due to syntax issues, etc. All screen widths / breakpoints should be reviewed.

### Scope / Impact analysis
Sometimes web components, if not formatted properly, can fail silently. This has been the case in content-build. We're not sure at this point whether it can happen in next-build, but special care should be taken to regression test the entirety of the affected pages.

Any code paths that are not covered in current production pages should be generated with mock data in a Tugboat for testing.

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
* **Functional testing:** 
* **Code review:** 
* **Design review:** 
* **Accessibility review:** 
* **Product acceptance:** 

## Acceptance criteria
_ACs should capture expected behavior, to inform test plans. Consider devices, documentation updates including KBs, change management, and [content model](https://prod.cms.va.gov/admin/structure/cm_document) when applicable._

- [ ] Contact information custom component is converted to DS web component ("Need help?")
- [ ] If any defects or functionality gaps (non-blocking or blocking) are identified in the web component, ticket(s) are added to the [DST issues list](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues)
- [ ] All code paths are covered by test URLs and present in the PR description with screenshots
- [ ] All screen widths are verified
- [ ] All affected R&S templates are verified from top to bottom for regressions
  - [ ] Checklist
  - [ ] FAQ Multiple Q&A
  - [ ] Media List Images (will require Tugboat content added for testing)
  - [ ] Media List Videos (will require Tugboat content added for testing)
  - [ ] Q&A
  - [ ] Step by Step
  - [ ] Resources & Support Detail Page
- [ ] Impact analysis is filled out and QA test plan is updated accordingly
- [ ] Does this product have an existing regression test plan?
	- [ ] Yes, link: 
	- [x] No, regression test plan will be created as part of the work / points in this ticket
- [ ] Design review
- [ ] Accessibility review

**Comment by randimays** [comment]
I recommended [in this Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1728405878592309?thread_ts=1727375505.418149&cid=C52CL1PKQ) that we wait to pick up this ticket until we are getting ready to ship Resources & Support templates into production in next-build, but deferring to @FranECross for that priority decision.

**Comment by FranECross** [comment]
@randimays I agree and marked this ticket blocked. I'll add a status at the top to that affect.

**Comment by FranECross** [comment]
@FranECross to create regression tests to capture existing behavior regarding the dynamic phone numbers, etc. See engineering notes.

@chriskim2311 Will you please check the points we assigned and add any other relevant info/change points as needed.
